.github/workflows/push.yml

```diff
@@ -164,4 +164,28 @@ jobs:
         user_email: '${{env.email}}'
         user_name: '${{github.actor}}'
         commit_message: 'Talawa API docs updated'
-      
+        
+# You can find the deployment instructions in the scripts/cloud-api-demo/README.md file
+  Deploy-Workflow:
+    name: Deploying Application to Cloud VPS
+    runs-on: ubuntu-latest
+    steps:
+      - name: Execute remote ssh commands 
+        uses: appleboy/ssh-action@v1.0.3 
+        env:
+          INPUT_ACCESS_TOKEN_SECRET: ${{ secrets.API_DEMO_ACCESS_TOKEN_SECRET }}
```

Comment:
Access tokens are generated above. Can you rename those and use them instead?

![image](https://github.com/PalisadoesFoundation/talawa-api/assets/16875803/85abba68-59f8-4e65-842e-711a9dcad4bc)

Reply:
Done